canary/e2e/features/liveness/face-detect.feature

```diff
@@ -0,0 +1,11 @@
+Feature: Liveness Start Screen
+
+  Test Liveness CDN
+
+  Background:
+    Given I'm running the example "/"
```

Comment:
Let's write an example specifically for liveness. Seems we are re-using [overloading] the sign-in example here?

Reply:
I think the goal here was to ensure that our library can build no matter which portions of the app you are using, @0618 maybe you can chime in here

Reply:
Think we should use a different example to run, like `ui/components/liveness/cdn` or something example `/` is used by authenticator. might be confusing to see sign-in and sing-up tests fail bc os liveness issue.

Reply:
@ioanabrooks does the path matter? If there's a cdn issue, face-detect would fail, but sign-in and sign-up should be fine, right?

Reply:
Does the page render okay in case of failure?

Reply:
We removed the Liveness component from the example so this test will fail. Add `@skip` tag or remove this file or this test will fail.